src/components/Input/input.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import "./input.css";
+
+const Input = ({ updateTask, printInput }) => {
+  const [input, setInput] = useState("");
+  const [colorInput, setColorInput] = useState(false);
+
+  const update = (event) => {
+    setInput(event.target.value);
+  };
+
+  const add = () => {
+    let value = input;
+    if (value === "") {
+      setColorInput(true);
+    } else {
+      setColorInput(false);
+      setInput("");
+      updateTask(value);
+    }
+  };
+
+  useEffect(() => {
+    setInput(printInput);
+  }, [printInput]);
```

Comment:
what is printInput? what is the use of it @PuniaMohit ?

Reply:
printInput contains todo task which is to be shown in input tag for editing.